recipes/sphinx-markdown-tables/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "sphinx-markdown-tables" %}
+{% set version = "0.0.12" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 08779e77832b6562b6b86e4270ae50b0b10010fdd35cf9aadcb5b8246571b933
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  noarch: python
+
+requirements:
+  host:
+    - markdown ==3.0.1
+    - pip
+    - python
+  run:
+    - markdown ==3.0.1
+    - python >=3.5
+    - recommonmark
+
+test:
+  imports:
+    - sphinx_markdown_tables
+  requires:
+    - markdown ==3.0.1
+    - recommonmark
+    - sphinx
+    - python =3
+  files:
+    - test.md
+
+about:
+  home: "https://github.com/ryanfox/sphinx-markdown-tables"
+  license: GPL-3.0-only
+  license_family: GPL3
+  license_file: LICENSE
```

Comment:
Since you are manually packaging the license file, please put a comment about this line with the following upstream issue as a reminder to check if the license file is being included in the source distribution in future releases:

https://github.com/ryanfox/sphinx-markdown-tables/issues/27

Reply:
Thanks. I have added a comment mentioning that the license_file is being manually included in the recipe and also the issue you raised.